### ISSUE TITLE
List of use cases/examples

### ISSUE THREAD

**Issue by stenskjaer** [scta/scta-community#4]
I would be nice, and very useful for us too, I think, if we could create a small list of examplary uses of the SCTA approach.
I imagine that would include links to examples of 
- Manuscript images in the context of a lbp-web text.
- Open annotation data fetched in to a specific paragraph of a lbp-web text.
- Example manuscripts to look at in the mirador viewer, explaining the construction of a view from disparate resources.
- Link to the print.lombardpress.org with explanation of the movement from xml to print page.

Others?

It would be cool for us to have such a page to pull up when explaining people about this, or to just point to it. And to new members, it makes it all a lot easier to grasp.

**Comment by uzahnd** [comment]
I fully agree that this would be very useful. A further example might bring together different presentations of the same data (i.e. a questionlist in scta and in rcs; or a paragraph in lbp and in magistersententiarum, etc)
